docs/development/developer_guide.md

```diff
@@ -5,6 +5,16 @@ Kubeflow Training Operator is currently at v1.
 ## Requirements
 
 - [Go](https://golang.org/) (1.22 or later)
+- [Docker](https://docs.docker.com/) 
+- [Docker Buildx](https://docs.docker.com/build/buildx/) 
```

Comment:
We usually build the multi-arch images. So, we need it.

Reply:
`docker buildx` CLI is included in Docker Desktop by default: https://docs.docker.com/build/architecture/#install-buildx, isn't ?
Btw, we don't use it today in Training Operator Makefile: https://github.com/kubeflow/training-operator/blob/master/Makefile#L90

Reply:
Just saw your message here: https://github.com/kubeflow/training-operator/pull/2074#discussion_r1578286126, thanks!

Reply:
> Btw, we don't use it today in Training Operator Makefile: https://github.com/kubeflow/training-operator/blob/master/Makefile#L90

Yeah, we should update it since we actually build images with multi-arch in CI.